metadata_tools/hosts/GO_0xxx/tests/test_geometry.py

```
@@ -2,13 +2,11 @@
 # GOSSI-specific metadata geometry unit tests
 ################################################################################
 import unittest
-
 import pdstable
-#import pdsparser
 import numpy as np
 
-import metadata_tools.util as util
-import metadata_tools.hosts.GO_0xxx.host_config as config
+#metadata_tools.util as util
```

Flake8 (7.3.0)

[error] block comment should start with '# '

(E265)